IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m

```diff
@@ -256,6 +257,8 @@ - (nullable instancetype)initWithGroup:(nullable NSString *)keychainGroup
             return nil;
         }
 
+        self.synchronizationQueue = dispatch_queue_create("com.microsoft.msidmackeychaintokencache", DISPATCH_QUEUE_CONCURRENT);
```

Comment:
what if there're multiple instances of Mac keychain token cache? we normally add NSUUID to the name of the queue for uniqueness.

Reply:
Since the queue is to protect the underlying OS keychain API I think you don't want it unique per token cache instance. Two queues shouldn't be able to write to the keychain at the same time.